README.md

```diff
@@ -100,5 +134,11 @@ Once in Vanilla JS, and once in jQuery, write a function that adds an event list
 
 Your Answer:
 ```js
-// write code here
-```
+
+document.querySelector("submit-quiz").addEventListener("click", submitQuiz);
+
```

Comment:
Not quite, first we still need to use `"."` before our class name when using `document.querySelector`. Also, we need a `$` in `jQuery` to pass in the selector:
```

// Vanilla JS
var button = document.querySelector(".submit-quiz");
button.addEventListener("click", function(){
  alert("Great job on Quiz 4!");
});

// jQuery
$(".submit-quiz").on("click", function(){
  alert("Great job on Quiz 4!");
});
```